flytekit/clients/auth/authenticator.py

```diff
@@ -224,12 +224,8 @@ def __init__(
 
     def refresh_credentials(self):
         resp = token_client.get_device_code(self._device_auth_endpoint, self._client_id, self._audience, self._scope)
-        print(
-            f"""
-To Authenticate navigate in a browser to the following URL: {resp.verification_uri} and enter code: {resp.user_code}
-OR copy paste the following URL: {resp.verification_uri_complete}
-        """
-        )
+        text = f"To Authenticate, navigate in a browser to the following URL: {click.style(resp.verification_uri, fg='blue', underline=True)} and enter code: {click.style(resp.user_code, fg='blue')}"
```

Comment:
Actually let's use rich console

Reply:
Do you mean using the [logger](https://github.com/flyteorg/flytekit/blob/e44b8027bd0f59c0f90eb5f86bfa181aebad0d74/flytekit/loggers.py#L18)? Looks kinda weird  
<img width="1443" alt="image" src="https://github.com/flyteorg/flytekit/assets/24364830/ec28f43d-319d-4caa-aff0-a4fbf9276f99">

Reply:
ya not the logger, i meant rich.console. but let it be what it was, can you revert and i can approve!

Reply:
i just reverted